examples/experimental/dagster-airlift/examples/tutorial-example/migrate.py

```diff
@@ -40,7 +40,9 @@ def _multi_asset() -> None:
 
 
 def export_duckdb_to_csv_defs(args: ExportDuckDbToCsvArgs) -> Definitions:
-    spec = AssetSpec(key=args.table_name, deps=["customers"])
+    spec = AssetSpec(
```

Comment:
ugly, but previous asset key was causing problems - can probably clean up

Reply:
Just explicitly pass in the asset spec?

I think that way it is obvious and the user has control